pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -43,6 +43,12 @@ let
       # Patch framework detection to support X.framework/X.tbd,
       # extending the current support for X.framework/X.
       ./qtbase.patch.d/0012-qtbase-tbd-frameworks.patch
+
+      (fetchurl {
```

Comment:
I have to use `fetchurl` instead of `fetchpatch` due to this issue: https://github.com/NixOS/nixpkgs/issues/182141.

Reply:
Please don't.  Probably commit it into nixpkgs repo.  The download will probably keep changing the hash over time, due to the way it's generated dynamically by GitHub.

Reply:
(that's the main motivation for `fetchpatch`, by the way)

Reply:
okay.jpeg